src/util/delayed-mount/delayed-mount.ts

```diff
@@ -0,0 +1,23 @@
+import { useEffect, useState } from 'react';
+
+export const useDelayedMount = (desiredState: boolean, delayTime: number) => {
+  const [mounted, setMounted] = useState(false);
+  const [visible, setVisible] = useState(false);
+
+  useEffect(() => {
+    let timeoutId;
```

Comment:
Specify type

Reply:
removed this variable, I don't know what was its purpose:P